Rakefile

```diff
@@ -151,6 +151,24 @@ namespace :build do
         HipChat.log 'Migrating <b>dashboard</b> database...'
         RakeUtils.rake 'db:migrate'
 
+        # Update the schema cache file, except for production which always uses the cache.
+        schema_cache_file = dashboard_dir('db/schema_cache.dump')
+        unless rack_env?(:production)
+          RakeUtils.rake  'db:schema:cache:dump'
+          if RakeUtils.file_changed_from_git?(schema_cache_file)
+            # Staging is responsible for committing the authoritative schema cache dump.
+            if rack_env?(:staging)
+              RakeUtils.system 'git', 'add', schema_cache_file
+              HipChat.log 'Committing updated schema_cache.dump file...', color: 'purple'
+              RakeUtils.system 'git', 'commit', '-m', '"Update schema cache dump after schema changes."'
```

Comment:
would it be safer to only commit that file instead of potentially scooping up random files on staging (people shouldn't be staging things they don't want to commit, but I could see that happening...)